File: pop/lab0/student.c
#include <stdio.h>
struct student
{
	int roll;
	char name[25];
};
void main()
{
	int n,i;
	printf("Enter number of students:");
	scanf("%d",&n);
	printf("Enter student details:\n");
	struct student a[n];
	for(i=0;i<n;i++)
	{
		printf("Student %d:\n",i+1);
		printf("Enter name:");
		scanf("%s",a[i].name);
		printf("Enter roll no:");
		scanf("%d",&a[i].roll);
	}
	printf("Printing details...\n");
	for(i=0;i<n;i++)
	{
		printf("Student %d\n",i+1);
		printf("Name = %s\nRoll no = %d\n",a[i].name,a[i].roll);
	}
}

File: pop/lab5/student.java
import java.util.Scanner;
class marks
{
		String name;
		float s1,s2,s3,s4,s5;
}
class student
{
	public static void main(String args[])
	{
		Scanner s =new Scanner(System.in);
		System.out.println("Enter number of students:");
		int n = s.nextInt();
		marks obj[] = new marks[n];
		for(int i=0;i<n;i++)
		{
			obj[i] = new marks();
			System.out.println("Enter name:");
			obj[i].name = s.next();
			System.out.println("Enter marks in 5 subjects:");
			obj[i].s1 = s.nextFloat();
			obj[i].s2 = s.nextFloat();
			obj[i].s3 = s.nextFloat();
			obj[i].s4 = s.nextFloat();
			obj[i].s5 = s.nextFloat();
		}
		for(int i=0;i<n;i++)
		{
			System.out.println("Name: "+obj[i].name+"\nSum: "+sum(obj[i]));
		}
	}
	static float sum(marks obj)
		{
			return obj.s1+obj.s2+obj.s3+obj.s4+obj.s5;
		}
}